src/services/search.js

```diff
@@ -4,11 +4,13 @@ import request from '../utils/request';
 export async function queryIndexMapping(params) {
   const { datastoreConfig, indices } = params;
 
-  const url = `${datastoreConfig.prefix}${datastoreConfig.run_index}${indices[0]}/_mappings`;
+  const mappingIndix = `${datastoreConfig.prefix}${datastoreConfig.run_index}${
```

Comment:
Oops; `mappingIndex`??

Reply:
yeah need to think something better

Reply:
Simply, mappings would be fine

Reply:
I was really just commenting on the typo. I wouldn't object to a more clear name, if anyone can think of one.